### PR TITLE
Reformat everything

### PR DESCRIPTION
**Pull request by lemaitre-aneo**
Also makes the pipeline fail if the formatting is not correct.

### CODE REVIEW

**Review by jfonseca-aneo — APPROVED** [review]

**Review by jfonseca-aneo — APPROVED** [review]

File: Adaptors/Memory/src/ArmoniK.Core.Adapters.Memory.csproj [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <IsPackable>true</IsPackable>
+    <Nullable>enable</Nullable>
```
Comment (aneojgurhem) [inline Adaptors/Memory/src/ArmoniK.Core.Adapters.Memory.csproj:12]: Why is this here ? It's reformatting pr !

Comment (lemaitre-aneo) [reply Adaptors/Memory/src/ArmoniK.Core.Adapters.Memory.csproj:12]: Because it forces resharper to use `?` instead of `[CanBeNull]`.
